storage/Utils/AppUtil.h

```diff
@@ -132,22 +134,18 @@ void classic(StreamType& stream)
      */
     struct NameSchema
```

Comment:
This class does not have a description and neither does format_to_name_schemata
What is this good for? Maybe a couple of examples will be enough?

Extracting a numeric/alphabetic id (or several ids) out of a device name? OK, but what is the padding good for?

Reply:
Both have a description. Anyway I can add an example for the usage.